app.js

```diff
@@ -3,11 +3,14 @@ const expressLayouts = require('express-ejs-layouts');
 const session = require('express-session');
 const favicon = require('serve-favicon');
 const database = require('./config/database');
+const indexController = require('./controllers/indexController');

```

Comment:
you can export all of these from an index.js file from the controller folder and import all of them in a single line like :
```javascript
const { indexController, cartController, etcController } = require('./controllers');
```

Reply:
Fixed